docs/developer/contributing/development-github.asciidoc

```diff
@@ -12,17 +12,19 @@ repo, which we'll refer to in later code snippets.
 [discrete]
 === Branching
 
-* All work on the next major release goes into master.
-* Past major release branches are named `{majorVersion}.x`. They contain
-work that will go into the next minor release. For example, if the next
-minor release is `5.2.0`, work for it should go into the `5.x` branch.
-* Past minor release branches are named `{majorVersion}.{minorVersion}`.
-They contain work that will go into the next patch release. For example,
-if the next patch release is `5.3.1`, work for it should go into the
-`5.3` branch.
-* All work is done on feature branches and merged into one of these
-branches.
-* Where appropriate, we'll backport changes into older release branches.
+At Elastic, all projects in the stack, including Kibana, are released at the same time with the same version number. Most of these projects have the following branching strategy:
+
+* `master` is the next major version.
+* `<major>.x` is the next minor version.
+* `<major>.<minor>` is the next release of a minor version, including patch releases.
+
+As an example, let's assume that the `7.x` branch is currently a not-yet-released `7.6.0`. Once `7.6.0` has reached a feature complete stage, it will be branched to `7.6` and `7.x` will be updated to reflect `7.7.0`. The release of `7.6.0` and subsequent patch releases will be cut from the `7.6` branch. At any time, you can verify the current version of a branch by inspecting the `version` attribute in the `package.json` file within the Kibana source.
+
+Pull requests are made into the `master` branch and then backported when it is safe and appropriate.
+
+* Breaking changes do not get backported and only go into `master`.
+* All non-breaking changes can be backported to the `<major>.x` branch.
+* Changes backported to a `<major>.<minor>` branch must be justified based on the severity and confidence of the change and the reasoning included in the pull request description.
```

Comment:
Really like the changes you made in this iteration.

I have one suggestion, but I'm torn on whether it's an improvement or not. Throwing it out there for other's thoughts.

```suggestion
* Features should not be backported to a `<major>.<minor>` branch.
* Bugs backported to a `<major>.<minor>` branch must be justified based on the severity of the bug, and the confidence of the change, and the reasoning should be included in the pull request description.
```

Reply:
I went back and forth and ended up with the general "changes" wording as I was thinking about what would not fit within the "feature" or "bug" category. But, reading through this now, I think it's good to be explicit. 

It also raised the idea for me; documentation changes can go into any branch at any time. I will push an update for that.